docs/Tutorials/Contracts/Deploying-Contracts.md

```diff
@@ -134,16 +132,21 @@ The Developer Quickstart provides an [example of a public transaction script](ht
     so we append that to the binary value).
 * `value` - amount of Ether/Wei transferred from the sender to the recipient.
 
-As the example demonstrates, once the transaction `tx` is created, you can sign it with the private
-key of the account. You can then serialize it and call `eth_sendSignedTransaction` to deploy the
-contract.
+Run the JavaScript code to send the transaction:
+
+```bash
+node public_tx.js
+```
+
+This example code creates the transaction `tx`, signs it with the private key of the account, serializes it, then calls
+`eth_sendSignedTransaction` to deploy the contract.
 
 ## Using `eth_sendTransaction`
 
-Use [`eth_sendTransaction`](https://eth.wiki/json-rpc/API) as an alternative to `eth_sendSignedTransaction`.
-However, Hyperledger Besu does not support the `eth_sendTransaction` API call and keeps account
-management separate for stronger security. Instead, Besu uses
-[EthSigner](https://docs.ethsigner.consensys.net/en/stable/) to make the `eth_sendTransaction` API call.
+You can use [`eth_sendTransaction`](https://eth.wiki/json-rpc/API) as an alternative to `eth_sendSignedTransaction`.
+However, Hyperledger Besu does not support the `eth_sendTransaction` API call and keeps account management separate for
+stronger security.
+Instead, Besu uses [EthSigner](https://docs.ethsigner.consensys.net/en/stable/) to make the `eth_sendTransaction` API call.
```

Comment:
I would say 
You can configure EthSigner with your Besu node to make the eth_sendTransaction call
or something like that - Besu doesn't know anything about EthSigner